src/plugins/brightcovePlugin.js

```diff
@@ -146,13 +147,27 @@ export default function createBrightcovePlugin(options = { concept: false }) {
         id={figureId}
         type={options.concept ? 'full-column' : 'full'}
         resizeIframe>
-        <iframe
-          title={`Video: ${brightcove.name}`}
-          aria-label={`Video: ${brightcove.name}`}
-          frameBorder="0"
-          {...getIframeProps(data, brightcove.sources)}
-          allowFullScreen
-        />
+        <div className="brightcove-video">
+          <iframe
+            className="original"
+            title={`Video: ${brightcove.name}`}
+            aria-label={`Video: ${brightcove.name}`}
+            frameBorder="0"
+            {...getIframeProps(data, brightcove.sources)}
+            allowFullScreen
+          />
+          <iframe
+            className="synstolket hidden"
+            title={`Video: ${brightcove.name}`}
+            aria-label={`Video: ${brightcove.name}`}
+            frameBorder="0"
+            {...getIframeProps(
+              { ...data, videoid: '6226280526001' },
```

Comment:
bruker en vilkårlig videoid for den "synstolka videoen" for å vise at videoen endres

Reply:
Ser at koden her forsøker å bruke reference_id, men det skal den ikkje gjøre lenger. 
Linja med `const id = ...` kan fjernes, og så kan brightcove.id brukes der id brukes i dag.

Då kan du bruke brightcove.reference_id som videoid i den skjulte iframen. Mulig du må fjerne 'ref:' fra den iden for at det skal bli korrekt.